monitoring/grafana/README.md

```diff
@@ -4,6 +4,17 @@ that will help you get started quickly introducing monitoring capabilities to yo
 
 Quick Start
 -----------
+Before generating the dashboards on your own, note that you may find the sample
+dashboard in importable JSON format inside the folder
+`grr/monitoring/grafana/grr_grafanalib_dashboards/dashboards_for_use`. From
+there you may import the sample dashboards without re-generating them to your
+Grafana instance. Only if you wish to generate the dashboards, proceed.
+
+**Please note the following edge case**: the panel "API Calls Count Rate by other statuses (not
+  SUCCESS)" inside the AdminUI dashboard will *not* show a spike in the graph
+  given a single HTTP API call, where the status of the call is not SUCCESS.

```

Comment:
I'd phrase it slightly differently: "given a single HTTP API call" -> "when there's only one data point present (only a single API call of a given method was made)".